### DISCUSSION TITLE
Is there something I'm missing recompiling CEFGlue or WebView for Avalonia 11 ?

### DISCUSSION THREAD

**Discussion by Ready4Next** [AvaloniaUI/Avalonia#9456] [Q&A] [answered]
Hi all,

I'm new to Avalonia, and I was wondering if someone have successfully built an updated version of those controls ? I have taken the source from their respective github locations, but failed to compile, both are saying the same thing which I don't understand:

WebView Avalonia:
'No precompiled XAML found for SampleWebView.Avalonia.App, make sure to specify x:Class and include your XAML file as AvaloniaResource'

CEFGlue:
No precompiled XAML found for Xilium.CefGlue.Demo.Avalonia.App, make sure to specify x:Class and include your XAML file as AvaloniaResource'

I have attached my modified version of WebView Avalonia.

What am I doing wrong / What do I forgot ?

Thanks.
[WebView-master.zip](https://github.com/AvaloniaUI/Avalonia/files/10025061/WebView-master.zip)

**Comment by kekekeks** [comment, marked as answer]
It's recommended to use [CefNet](
https://github.com/CefNet/CefNet) since it's properly updated to latest stable Avalonia, has better platform support and fewer issues in general.

**Reply by Ready4Next** [reply]
Thank you for your quick answer, unfortunately, I haven't been able to compile it successfully yet. I will try again later.

**Reply by bogatkov** [reply]
The [CefNet](https://github.com/CefNet/CefNet) project seems to be deleted.
